bundles/org.openhab.binding.plex/README.md

```diff
@@ -0,0 +1,172 @@
+        <SyncList itemsCompleteCount="0" totalSize="0" version="2"/>
+        <Connection uri="http://192.168.1.220:32500"/>
+    </Device>
+    <Device name="MacBook-Pro.local" publicAddress="XXX.XXX.XXX.XXX" product="Plex for Mac" productVersion="1.41.0.2876-e960c9ca" platform="osx" platformVersion="12.2" device="" model="standalone" vendor="" provides="client,player,pubsub-player" clientIdentifier="5ehipgz2ca60ikqnv9jrgojx" version="1.41.0.2876-e960c9ca" id="507110703" token="XXX" createdAt="1625081186" lastSeenAt="1647973509" screenResolution="1680x1050,1680x1050" screenDensity=""> </Device>
+</MediaContainer>
+```
+
+Find the `Device` block of the player you want to add and fill in the `clientIdentifier` as `playerID`
+
+| Name        | Type    | Description                                                                                | Default | Required | Advanced |
+|-------------|---------|--------------------------------------------------------------------------------------------|---------|----------|---------|
+| playerID    | text    | The unique identifier of the player. `clientIdentifier` from [https://plex.tv/devices.xml] | N/A     | yes      | no      |
+
+## Channels
+
+The PLEX Player supports the following channels:
+
+| Channel              | Type   | Read/Write | Description                                                    |
+|----------------------|--------|------------|----------------------------------------------------------------|
+| currentPlayers       | String | RO         | The count of players currently configured to watch on PLEX     |
+| currentPlayersActive | String | RO         | The count of players currently being used on PLEX              |
+| state                | String | RO         | The current state of the Player                                |
+| power                | String | RO         | The power status of the player                                 |
+| title                | String | RO         | The title of media that is playing                             |
+| type                 | String | RO         | The current type of playing media                              |
+| endtime              | String | RO         | Time at which the media that is playing will end               |
+| progress             | Dimmer | RO         | The current progress of playing media                          |
+| art                  | String | RO         | The URL of the background art for currently playing media      |
+| thumb                | String | RO         | The URL of the cover art for currently playing media           |
+| player               | String | RW         | The control channel for the player `PLAY/PAUSE/NEXT/PREVIOUS`  |
```

Comment:
This could be of type `Player`. See https://www.openhab.org/docs/configuration/items.html#type